docs/integrations/config-checkers/index.md

```diff
@@ -1 +1,22 @@
 # Configuration Checkers
+
+Starboard relies on a configuration checker to run variety of checks  on discovered Pods and controllers to make sure
```

Comment:
"controllers" is not very descriptive in this context, maybe "workloads"?

Reply:
Changed to workloads